.github/workflows/test-action.yaml

```diff
@@ -0,0 +1,26 @@
+name: Tests
```

Comment:
rename the file. `test-action` is bad name

Reply:
`+`